src/component/base/DraftEditor.react.js

```diff
@@ -190,6 +191,10 @@ class DraftEditor extends React.Component {
       if (!this.props.readOnly) {
         const method = this._handler && this._handler[eventName];
         method && method(this, e);
+      } else if (eventName === 'onCopy') {
+        editOnSelect(this)
```

Comment:
Why is this call necessary? This reconciles selection between the editor state and the DOM.